### DISCUSSION TITLE
Package updates

### DISCUSSION THREAD

**Discussion by vadimffe** [Live-Charts/LiveCharts2#451] [General]
Just as a comment. Could packages be updated more often with current versions. There are currently some crucial updates in repo already that would be nice to have. This would help to test current version in real case scenarios and spot other bugs if there are any. I know I can pull latest version from repo and merge with my fork, but then I find it more convenient to use NuGet package with updating features and all the other benefits. (or am I missing something?)

For example in my app I am currently having:
`<PackageReference Include="LiveChartsCore.SkiaSharpView.XamarinForms" Version="2.0.0-beta.101" /> 
`
However it is missing some recent updates that are already in this repo.

Also in repo it reads: `dotnet add PROJECT package LiveChartsCore.SkiaSharpView.XamarinForms --version 2.0.0-beta.100`

**Comment by beto-rodriguez** [comment]
Yes, the current master branch has a lot of fixes, I will publish a new release asap.

Sorry for the delay, this release includes a lot of cool features, such as a huge improvement to zooming and panning in touch devices, 
adds support for Uno platform, and Eto Forms, and the new web site https://lvcharts.com

I am getting everything ready for the release, I hope that in the worst of the cases the release is on monday.

**Reply by vadimffe** [reply]
I have updated version by just changing it in Project file:

`    <PackageReference Include="LiveChartsCore.SkiaSharpView.XamarinForms" Version="2.0.0-beta.200" />
`

Just a side comments:

Following this guide: https://lvcharts.com/docs/Xamarin/2.0.0-beta.220/Overview.Installation
It says: `Install-Package : Unable to find package 'LiveChartsCore.SkiaSharpView.XamarinForms'`

Also here: https://github.com/beto-rodriguez/LiveCharts2/packages/896936
It says beta 100

**Reply by beto-rodriguez** [reply]
They are updated now
